content/en/docs/ops/configuration/traffic-management/dns-proxy/test.sh

```diff
@@ -33,6 +33,13 @@ _verify_first_line snip_dns_capture_in_action_3 "$snip_dns_capture_in_action_3_o
 snip_address_auto_allocation_1
 _verify_contains snip_address_auto_allocation_2 "*   Trying 240.240."
 
+# verify external tcp services without vips
+snip_external_tcp_services_without_vips_1
+snip_external_tcp_services_without_vips_2
+snip_external_tcp_services_without_vips_3
+snip_external_tcp_services_without_vips_4
+_verify_like snip_external_tcp_services_without_vips_5 "$snip_external_tcp_services_without_vips_5_out"
```

Comment:
`_verify_lines` is only tolerant of varying IPs if it is a space-separated token. In this case it won't work because the token is `ADDRESS=x.x.x.x`. You could fix it by changing the output format to make it a token, but I think maybe it's better to go back to your previous implementation using `_verify_lines` instead:
```suggestion
_verify_lines snip_external_tcp_services_without_vips_5 "
+ outbound|9000||tcp-echo.external-2.svc.cluster.local
+ outbound|9000||tcp-echo.external-1.svc.cluster.local
"
```